spring-mvc-simple-2/src/main/java/com/baeldung/flash_attributes/model/Poem.java

```diff
@@ -0,0 +1,39 @@
+package com.baeldung.flash_attributes.model;
+
+import org.apache.logging.log4j.util.Strings;
+
+public class Poem {
+    private String title;
+    private String author;
+    private String body;
+
+    public static boolean isValidPoem(Poem poem) {
+        return poem != null && Strings.isNotBlank(poem.getAuthor()) && Strings.isNotBlank(poem.getBody())
+                && Strings.isNotBlank(poem.getTitle());
```

Comment:
Only need to indent two spaces from where the statement begins.

Reply:
Addressed